xmtp/src/conversations.rs

```diff
@@ -191,6 +180,8 @@ impl<A: XmtpApiClient> Conversations<A> {
 
         //TODO: Validate message
 
+        // TODO move this logic into a Conversation::save_message() method
+        SecretConversation::ensure_conversation_exists(client, conn, &message_obj.convo_id)?;
```

Comment:
We're not guaranteed to have received an invite first (which would have created the conversation row in the DB), so we need to add the row when receiving messages.